css_files/home_page.css

```diff
@@ -0,0 +1,156 @@
+@import "./product_page.css";
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  
+}
+
+html{
+  font-size: 62.5%;
+}
+
+.app{
+  grid-template-columns: 100%;
+  grid-template-areas: "navbar"
+  "content"
+  "footer"
+  ;
+}
+
+
+body{
+    font-family: 'Lato', sans-serif;
+    color: var(--black-shade);
+    background-color: var(--bg-color-white);
+}
+
+.navbar{
+    grid-area: navbar;
+  }
+
+.header-left{
+    display: flex;
+    align-items: center;
+    gap: var(--sm-space);
+}
+  
+.searchbar{
+    margin: auto;
+    width: 30%;
+}
+  
+.input-primary{
+    width: 100%;
+}
+
+.btn-badge{
+    background-color: white;
+    color: var(--primary-color);
+    z-index: 9;
+  }
+  
+.badge-wrapper-i,.icons{
+    cursor: pointer;
+    font-size: 2.4rem;
+    color: var(--pure-white);
+}
+
+.content{
+    width: 100%;
+    margin: auto;
+}
+
+.hero{
+    color:var(--black-shade);
+    height: 100%;
+}
+
+.prime-heading{
+  color: var(--primary-color);
+}
+
+.btn-hero{
+  font-size: 2rem;
+  display: block;
+  width: fit-content;
+}
+
+.hero-left-content{
+  margin-top: 10rem;
+  width: fit-content;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
One suggestion, you can use shorthand for this like, margin: 10rem auto 0; am I right?